.github/update_templates.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#
+# This script updates the current repository with the latest version of the templates. It creates a new branch and a pull request.
+#
+
+function ensure_dependencies_or_exit() {
+  if ! command -v gh &> /dev/null; then
+    echo "gh is not installed. Please install it from https://cli.github.com/"
+    exit 1
+  fi
+}
+
+function ensure_repo_preconditions_or_exit() {
+  # ensure main branch
+  if [ "$(git branch --show-current)" != "main" ]; then
+    echo "The current branch is not main. Please switch to the main branch."
+    exit 1
+  fi
+
+  # ensure a clean working directory
+  if [ -n "$(git status --porcelain)" ]; then
+    echo "The working directory is not clean. Please use a clean copy so no unintended changes are merged."
+    exit 1
+  fi
+}
+
+function create_and_show_pr_for_init_branch() {
+  local branch_name=$1
+
+  if git branch | grep -q "$branch_name"; then
+    git checkout "$branch_name"
+
+    title=$(head -n1 pr-description.md)
+    body=$(tail -n2 pr-description.md)
+
+    rm pr-description.md
+    git add .
+    git commit -m "remove the PR description"
+    git push
+
+    gh pr create --title "$title" --body "$body" --base main --head "$branch_name"
+    gh pr view --web
+  fi
+}
+
+ensure_dependencies_or_exit
+ensure_repo_preconditions_or_exit
+
+latest_template_path=$(mktemp -d -t repository-template-XXXXX)
+new_branch_name=$(basename "$latest_template_path")
+
+# clone the default branch to get the latest version of the template files
+gh repo clone https://github.com/Hapag-Lloyd/Repository-Template-Maven.git "$latest_template_path"
```

Comment:
This references the wrong repo. Should be https://github.com/Hapag-Lloyd/Repository-Template-Terraform-Module, right?